### ISSUE TITLE
Element timing/LCP: Replace `TAO` restriction with coarsening

### ISSUE THREAD

**Issue by noamr** [mozilla/standards-positions#1128]
## Request for Mozilla Position on an Emerging Web Specification

* Specification title: Element timing/LCP: Replace `TAO` restriction with coarsening
* Specification or proposal URL (if available):  https://w3c.github.io/largest-contentful-paint/
* Explainer URL (if available): https://github.com/w3c/paint-timing/blob/main/presentation-timestamps.md and [this document](https://docs.google.com/document/d/1VxgMf1wlWzB4ViAW4ohkOe3AT0wQZKk7hC3IVq-cuw0/edit?tab=t.0#heading=h.fmic3y1ir4).
* Proposal author(s) (`@`-mention GitHub accounts): @noamr @mmocny
* Caniuse.com URL (optional): 
* Bugzilla URL (optional): 
* Mozillians who can provide input (optional): @BasMoz @sefeng211
* WebKit standards-position: 

### Other information
This has been discussed at the WebPerfWG, see [minutes](https://docs.google.com/document/d/1ZWAUJZBJUvSUyShvKXNEU-cuCe47jpgbR69ckWZUTfI/edit?tab=t.0)

**Comment by sefeng211** [comment]
Hi Noam, I wasn't sure how this was discussed in the call, have we talked explicitly about removing it?

**Comment by noamr** [comment]
> Hi Noam, I wasn't sure how this was discussed in the call, have we talked explicitly about removing it?

Yes, the linked minutes specifically point out removing the TAO restriction, also in the [prior WebPerfWG calll](https://docs.google.com/document/d/1tw9QTHWvXg-loG6qaeeosOXTCeH41wst6MehQsc3WwM/edit?tab=t.0#heading=h.ex3ms275azam). However sometimes in minuted discussions things are not as clear after a while. In any case, that is the intention ATM and happy to discuss agian.
